app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectFragment.kt

```diff
@@ -317,19 +476,21 @@ class DeviceConnectFragment : Fragment(), IDeviceConnectView {
             if (p1 != null) {
                 if (p1.action.equals(WifiManager.SCAN_RESULTS_AVAILABLE_ACTION)) {
                     val wifiList = mainWifi.getScanResults()
-                    Timber.d("Check %s", checkDevice)
+                    Timber.e("Check %s", checkDevice)
                     if (checkDevice)
                         deviceConnectPresenter.availableDevices(wifiList)
-                    else
+                    else {
+                        stopProgress()
                         deviceConnectPresenter.availableWifi(wifiList)
+                    }
                 } else if (p1.action.equals(WifiManager.SUPPLICANT_STATE_CHANGED_ACTION)) {
                     Timber.d("Wifi changes")
                     if (p1.getParcelableExtra<Parcelable>(WifiManager.EXTRA_NEW_STATE) == SupplicantState.COMPLETED) {
                         Timber.d("Wifi Changes #2")
                         val wifiInfo = mainWifi.connectionInfo
                         if (wifiInfo != null) {
                             val ssid = wifiInfo.ssid
-                            Timber.d(ssid)
+                            Timber.e(ssid)
```

Comment:
same

Reply:
The message should be understandable like `"WiFi ssid: " + ssid`

Reply:
And why is it Timber.e

Reply:
Already changed that to d in the last commit.

Reply:
I think it's updated